exercises/audio tinkering 1/Music.py

```diff
@@ -0,0 +1,40 @@
+import pygame
+pygame.init()
+pygame.mixer.init()
+#set window dimensions and caption
+
+
+
+window = pygame.display.set_mode((800,600))
+pygame.display.set_caption("Jungle run")
+
+#load music, show file path
+sound = "C:\sprites\jungle-run-01."
+#load pygame mixer
+pygame.mixer.music.load("C:\sprites\jungle-run-01.MP3")
```

Comment:
Avoid declaring a literal here --- declare a variable and set the path as its value. Avoid the use of absolute paths as it affects portability.